src/test/kotlin/KotlinTestAzure.kt

```diff
@@ -96,7 +96,9 @@ fun doChatAzure(stream: Boolean, async: Boolean) {
 
     // Loads the API key from the .env file in the root directory.
     val key = dotenv()["OPENAI_TOKEN"]
-    val openai = AzureOpenAI(key)
+    val modelName = dotenv()["OPENAI_MODEL"]
+    val azureBaseUrl = dotenv()["OPENAI_AZURE_BASE_URL"]
+    val openai = AzureOpenAI(key, modelName = modelName, azureBaseUrl = azureBaseUrl)
```

Comment:
Errors like this should definitely have a check in the constructor, or the arguments should be redone so the azure arguments come before the optional ones. A builder class is also an option.